packages/prisma/migrations/20231113202947_add_ical_columns_to_booking/migration.sql

```diff
@@ -0,0 +1,3 @@
+-- AlterTable
+ALTER TABLE "Booking" ADD COLUMN     "iCalSequence" INTEGER NOT NULL DEFAULT 0,
+ADD COLUMN     "iCalUID" VARCHAR DEFAULT '';
```

Comment:
@joeauyeung In the case where you want this field to be unlimited, I don't think we need to go with VARCHAR since TEXT has already been the standard. My suggestion was to consider putting a max length on this field since it's not just free text.

Reply:
I'll change this back to TEXT. Although there's no limit on the length of the iCalUID I don't want to set an upper limit just in case there's a calendar provider that sets long ids.